e2e/karma/dummy_test.sh

```diff
@@ -0,0 +1,2 @@
+echo "Just a dummy test so that we have a test target for //... on bazelci ubuntu with bazel_integration_test"
```

Comment:
I don't get this - is there nothing to `bazel test` in this repo? nothing we need to assert?

Reply:
Nothing when you filter out the chrome & firefox tests and `bazel test ...` errors out if it doesn't find any matching tests